packages/remix/test/fixtures/01-remix-basics/app/b.server.ts

```diff
@@ -8,10 +8,13 @@ import { json } from '@remix-run/node';
 
 export async function loader() {
   const hi = await new Promise<string>((resolve, reject) => {
-    exec('echo hi from the B page', (err, stdout) => {
-      if (err) return reject(err);
-      resolve(stdout);
-    });
+    exec(
+      `echo hi from the B page running in ${process.env.VERCEL_REGION}`,
```

Comment:
**question (non-blocking):** Why spawn `echo`? If Node already sees `process.env.VERCEL_REGION`, then I'd think you can just resolve it, or use `setImmediate()` if you need to force async.

Reply:
The point of this test was to verify that `spawn` could be used in a Node route (and wouldn't subsequently cause the Edge bundle to fail).

Reply:
Ah, that makes sense. Is that worth mentioning in a comment?